airflow/operators/python.py

```diff
@@ -433,7 +433,8 @@ def execute_callable(self):
             if not isinstance(self.requirements, str):
                 requirements_file_contents = "\n".join(str(dependency) for dependency in self.requirements)
             else:
-                requirements_file_contents = self.requirements
+                with open(self.requirements, "r") as file:
+                    requirements_file_contents = "\n".join(file.readlines())
```

Comment:
Perhaps we can do this:

```python
if not isinstance(self.requirements, str):
    ...
elif os.path.exists(self.requirements):
    with open(self.requirements, "r") as f:
        ...
else:
    requirements_file_contents = self.requirements
```

This helps keep compatibility to the “incorrect” use of providing one single requirement as the argument.